forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/ApplicationAuditListener.java

```diff
@@ -82,6 +82,9 @@ protected Application prepareApplicationAudit(DelegateExecution execution) {
                 submittedBy = "Anonymous-user";
             }
         }
+        else {
+            submittedBy = "service-account-forms-flow-bpm";
```

Comment:
Doesn't this come as username from the authentication ?

Reply:
Task is not performed by user. May be a event(timer.. ) can make a submission. In this case authentication is null.